CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 💪 1.10.2
+
+- ### Fixes
+
+  - **Fixes bug preventing `wrangler publish` from deleting over 10000 keys - [ashleymichal], [issue/1398] [pull/1400]**
```

Comment:
need to add links here for the PRs, issues, and author - otherwise these won't go anywhere